conans/model/build_info.py

```diff
@@ -79,7 +79,7 @@ def __getitem__(self, item):
         return self._dependencies[item]
 
     def __repr__(self):
-        return TXTGenerator(self).content
+        return TXTGenerator(self, CppInfo(os.getcwd())).content
```

Comment:
I was looking at how this was used.  Originally, I had `None` in there, but that felt "more wrong" than putting in a dummy CppInfo object.  I wasn't entirely sure what the best way to refactor it would be.  I'm fine with making the change in this pull request, but I'll need some guidance on what you'd like me to do.